xivo/http_helpers.py

```diff
@@ -106,19 +108,28 @@ def _to_dict(self, headers):
         return dict(headers)
 
 
-def _log_request(url, response):
+def _log_request(url, response, hidden_fields=None):
     current_app.logger.info(
         'response: (%s) %s %s %s',
         request.remote_addr,
         request.method,
         url,
         response.status_code,
     )
+    params = {}
+    if (
+        response.data
+        and response.headers.get('Content-Type') not in NOT_PRINTABLE_CONTENT_TYPES
+    ):
+        params['body'] = BodyFormatter(response.data, hidden_fields)
+        fmt = "response body: %(body)s"
+    else:
+        fmt = "response body empty"
```

Comment:
3 different messages.
"response body: %(body)s"
"response body: empty"
"response body: not printable $(content_type)s"

getting `response body empty` can also be misleading if the body is not empty.

Reply:
I totally agree, that was my first implementation :) Thank you!